### PR TITLE
update subgraph url

### PR DESCRIPTION
**Pull request by buchaoqun**
**NOTE**

#### Please enable "Allow edits by maintainers" while putting up the PR.

---

> - If you would like to add a `volume` adapter please submit the PR [here](https://github.com/DefiLlama/adapters).
> - If you would like to add a `liquidations` adapter, please refer to [this readme document](https://github.com/DefiLlama/DefiLlama-Adapters/tree/main/liquidations) for details.

1. Once your adapter has been merged, it takes time to show on the UI. If more than 24 hours have passed, please let us know in Discord.
2. Sorry, We no longer accept fetch adapter for new projects, we prefer the tvl to computed from blockchain data, if you have trouble with creating a the adapter, please hop onto our discord, we are happy to assist you.
3. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
4. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
5. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
6. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/solv-protocol-funds exports TVL: 
        
 
 
    polygon-borrowed          0
    polygon                   0
    bsc                       176.80608669242477
    bsc-borrowed              212.07 k
    mantle-borrowed           2.41 M
    mantle                    3.88 M
    arbitrum                  105.39 M
    arbitrum-borrowed         8.91 M
    merlin                    87.3613
    ethereum                  552.004
    ethereum-borrowed         -9.99872
    merlin-borrowed           0
    borrowed                  11.52 M
    
    total                    109.27 M

**Comment by llamatester** [comment]
The adapter at projects/solv-protocol-funds exports TVL: 
        
 
 
    polygon-borrowed          0
    polygon                   0
    arbitrum-borrowed         0
    mantle-borrowed           0
    bsc                       174.6551306364136
    merlin                    86.2251
    mantle                    3.83 M
    merlin-borrowed           0
    ethereum                  541.8649999999999
    ethereum-borrowed         -10.00871872
    arbitrum                  104.40 M
    bsc-borrowed              209.49 k
    borrowed                  209.48 k
    
    total                    108.23 M

File: projects/Chocoinu/index.js
Comment (g1nt0ki) [inline projects/Chocoinu/index.js:1]: change to solv is fine, can you revert the deletion here?

**Comment by llamatester** [comment]
The adapter at projects/solv-protocol-funds exports TVL: 
        
 
 
    polygon-borrowed          0
    polygon                   0
    arbitrum-borrowed         0
    mantle-borrowed           0
    bsc                       173.94981713897738
    bsc-borrowed              208.65 k
    merlin                    85.865
    mantle                    3.80 M
    ethereum                  541.025
    ethereum-borrowed         -10.00871872
    merlin-borrowed           0
    arbitrum                  104.38 M
    borrowed                  208.64 k
    
    total                    108.19 M

**Comment by llamatester** [comment]
The adapter at projects/solv-protocol-funds exports TVL: 
        
 
 
    polygon-borrowed          0
    polygon                   0
    arbitrum-borrowed         0
    mantle-borrowed           0
    bsc                       173.98733381437293
    bsc-borrowed              208.69 k
    merlin                    86.24199999999999
    merlin-borrowed           0
    mantle                    3.81 M
    ethereum                  541.4805
    ethereum-borrowed         -10.00871872
    arbitrum                  104.40 M
    borrowed                  208.68 k
    
    total                    108.21 M

File: projects/Chocoinu/index.js
Comment (buchaoqun) [reply projects/Chocoinu/index.js:1]: Thanks.

**Pull request merged**